src/pages/workspace/WorkspacesListPage.tsx

```diff
@@ -106,30 +84,29 @@ function dismissWorkspaceError(policyID, pendingAction) {
     throw new Error('Not implemented');
 }
 
-function WorkspacesListPage({policies, allPolicyMembers, reimbursementAccount, userWallet}) {
+function WorkspacesListPage({
+    policies = {},
+    allPolicyMembers = {},
+    reimbursementAccount = {},
+    userWallet = {
+        currentBalance: 0,
+    },
```

Comment:
Most likely these default values aren't needed, in Ts we have optional chaining which should handle these gracefully

Reply:
I just removed all default values